package.json

```diff
@@ -13,10 +13,13 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "jest": "^26.0.1",
     "nodemon": "^2.0.2"
   },
   "scripts": {
     "start": "node index.js",
-    "dev": "nodemon index.js"
+    "dev": "nodemon index.js",
+    "jest": "jest",
```

Comment:
You might consider using the common script name `test`, e.g. `yarn test`.

Reply:
I almost went with that as the default. I'll change it back.